package/firmware/intel-microcode/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=intel-microcode
-PKG_VERSION:=20200616
-PKG_RELEASE:=1
+PKG_VERSION:=20210608
+PKG_RELEASE:=2
```

Comment:
I think PKG_RELEASE should increase only if something is changed on the same version.

Reply:
Just to match the debian upstream

Reply:
The PKG_RELEASE variable is OpenWrt specific and unrelated to upstream. We update it when some changes are done to a package without change the version number like adding an extra patch.

Reply:
Please check the new commit, I found that debian will only keep the latest version of the non-release version of the file, switch to the file with the release suffix, cancel the use of variables to get the file, and avoid the failure of upstream links